### ISSUE TITLE
AI Lawers not sueing other Lawers or Corporate Branches

### ISSUE THREAD

**Issue by LynxAbraxas** [civctp2/civctp2#216]
Issue to keep track of https://github.com/civctp2/civctp2/pull/186#issuecomment-536345303 now that #186 is merged.
The AI does not use its Lawers to sue Lawers or Corporate Branches of other civs.

**Comment by LynxAbraxas** [comment]
@MartinGuehmann could that be due to inappropriate configuration under:
https://github.com/civctp2/civctp2/blob/7a71370696827b4b9b10e2d7c4680febbe6416ec/ctp2_data/default/aidata/Goals.txt#L1792-L1825
in particular:
https://github.com/civctp2/civctp2/blob/7a71370696827b4b9b10e2d7c4680febbe6416ec/ctp2_data/default/aidata/Goals.txt#L1821
Should that be adjusted to be similar to e.g.
https://github.com/civctp2/civctp2/blob/7a71370696827b4b9b10e2d7c4680febbe6416ec/ctp2_data/default/aidata/Goals.txt#L724-L754

**Comment by MartinGuehmann** [comment]
> @MartinGuehmann could that be due to inappropriate configuration under:

No, Lawers are stealth and special units, so that is all right.

> in particular:
> https://github.com/civctp2/civctp2/blob/7a71370696827b4b9b10e2d7c4680febbe6416ec/ctp2_data/default/aidata/Goals.txt#L1821
> 

Lawyers and Corporate Branches are special attack units, so the threaten type should be right. I am actually not sure what this does if anything. But that would be something to look up in the code.

Actually, I think that the problem is that the Layers prefer to stop production in other cities. And since this is an attack you have to do again and again each turn, the Lawyers will just do that.

**Comment by LynxAbraxas** [comment]
> Actually, I think that the problem is that the Layers prefer to stop production in other cities. And since this is an attack you have to do again and again each turn, the Lawyers will just do that.

Hm, not sure if that is the problem because I have a lawyer now standing right beside a corporate branch of mine for many turns and far from any foreign city it could attack and go back.

**Comment by LynxAbraxas** [comment]
To rule out that #176 lead to these problems I tested what happens with a lawer conered by foreign corporate branches using 82a15c92: Still same problem, the lawer just walks off in the wild.
@MartinGuehmann how could I disable the other possible orders (like stop production in other cities) of lawers for testing?

**Comment by MartinGuehmann** [comment]
Well, you could disable the other orders by reducing their priorities. Make them negative for example. You should also switch on the debug army text to see what orders your Lawyers are following. For that check the chat window code that checks for Easter eggs.

**Comment by LynxAbraxas** [comment]
Test save-game:  [dbgSue.zip](https://github.com/civctp2/civctp2/files/3936133/dbgSue.zip).
